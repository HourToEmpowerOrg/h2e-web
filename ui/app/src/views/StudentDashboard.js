import React from 'react';
import ScheduleSection from '../components/sections/ScheduleSection';
import PrefferencesSection from '../components/sections/PreferencesSection';
import SeessionHistorySection from '../components/sections/SessionHistorySection';

class StudentDashboard extends React.Component {
  render() {
    return (
      <React.Fragment>
            <ScheduleSection></ScheduleSection>
            <SeessionHistorySection></SeessionHistorySection>
      </React.Fragment>
    );
  }
}

export default StudentDashboard;