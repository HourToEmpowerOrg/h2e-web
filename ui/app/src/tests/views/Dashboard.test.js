import React from 'react';
import { shallow, mount } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import StudentDashboard from '../../views/StudentDashboard'



const Container = StudentDashboard;

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Container />);
});

describe('StudentDashboard', () => {
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('should render both page headers <h4>', () => {
    const wrapper = mount(
      <MemoryRouter>
        <StudentDashboard/>
      </MemoryRouter>
    );
    expect(wrapper.find('h4').length).toBe(2);
  });
});