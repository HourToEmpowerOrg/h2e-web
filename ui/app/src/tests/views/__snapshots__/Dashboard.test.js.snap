// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StudentDashboard renders correctly 1`] = `ShallowWrapper {}`;
